### PR TITLE
Add support for ST3

### LINKED ISSUE #9: Support ST3
Is there a reason for not supporting ST3 ?

### PR DESCRIPTION
**Pull request by AmjadHD**
Fixes #9

### CODE REVIEW

**Comment by AmjadHD** [comment]
I forgot one of the maintainer must create an st3 branch first, cc @jfcherng .

File: LSP-pyright.sublime-settings [hunk 1]
```diff
@@ -12,47 +14,54 @@
...
-			// "reportUnusedImport": "information",
-			// "reportUnusedVariable": "information",
+		"python": {
+			"analysis": {
+				// Offer auto-import completions.
+				"autoImportCompletions": true,
+				// Automatically add common search paths like 'src'?
+				"autoSearchPaths": true,
+				// Additional import search resolution paths
+				"extraPaths": [
```
Comment (jfcherng) [inline LSP-pyright.sublime-settings:24]:
```json

		"python.analysis.extraPaths": [
			// For Sublime Text plugin development
			"$sublime_py_files_dir",
			"$packages",
		],
```

is removed here?

Comment (AmjadHD) [reply LSP-pyright.sublime-settings:24]: yes LSP3 does not support them iirc.

Comment (jfcherng) [reply LSP-pyright.sublime-settings:24]: But this removal changes the current behavior of ST 4.

**Comment by jfcherng** [comment]
> I forgot one of the maintainer must create an st3 branch first, cc @jfcherng .

Done!

**Comment by jfcherng** [comment]
We also need to make a PR to Package Control to for the `st3` branch.

**Review by rchl — APPROVED** [review]

**Comment by jfcherng** [comment]
Thanks!

**Pull request merged**